Projects/OpenXRTest/project.json.bak0

```diff
@@ -0,0 +1,24 @@
+{
+    "project_name": "OpenXRTest",
+    "project_id": "{947211d5-689a-437f-8ad7-7f558edcd40a}",
+    "origin": "The primary repo for OpenXRTest goes here: i.e. http://www.mydomain.com",
```

Comment:
do we want to set something here for github/o3de/o3de-extras or whatever it is?

Reply:
We already have a project.json: https://github.com/o3de/o3de-extras/blob/development/Projects/OpenXRTest/project.json

This file needs to be removed from this PR, but it is a good point.  We should update the real `project.json` for this project because it has default properties same as this file.